app/src/main/res/layout/layout_safe_item.xml

```diff
@@ -8,44 +8,131 @@
         android:layout_height="wrap_content"
         android:layout_alignParentEnd="true"
         android:layout_alignParentRight="true"
-        android:layout_centerVertical="true"
-        android:background="@drawable/selectable_background"
-        android:padding="8dp"
-        app:srcCompat="@drawable/ic_share_24dp" />
+        android:background="@drawable/safe_owner_background"
+        android:orientation="horizontal"
+        android:padding="4dp"
+        android:visibility="gone"
+        tools:visibility="visible">
+
+        <ImageView
+            android:layout_width="14dp"
+            android:layout_height="14dp"
+            android:scaleType="fitCenter"
+            android:src="@drawable/ic_owner" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="4dp"
+            android:layout_marginStart="4dp"
+            android:text="@string/owner"
+            android:textAllCaps="true"
+            android:textColor="@color/white"
+            android:textSize="9sp"
+            tools:ignore="SmallSp" />
+    </LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/layout_safe_item_authorizations_container"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignLeft="@+id/layout_safe_item_name_container"
+        android:layout_alignStart="@+id/layout_safe_item_name_container"
+        android:layout_below="@id/layout_safe_item_name_container"
+        android:layout_marginTop="16dp"
+        android:gravity="center"
+        android:orientation="horizontal">
+
+        <ImageView
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:src="@drawable/ic_authenticate_small_dark" />
+
+        <TextView
+            android:id="@+id/layout_safe_item_authorizations"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="4dp"
+            android:layout_marginStart="4dp"
+            android:text="-"
+            android:textSize="13sp"
+            tools:ignore="HardcodedText"
+            tools:text="3/5" />
+    </LinearLayout>
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignTop="@+id/layout_safe_item_authorizations_container"
+        android:layout_marginLeft="16dp"
+        android:layout_marginStart="16dp"
+        android:layout_toEndOf="@+id/layout_safe_item_authorizations_container"
+        android:layout_toRightOf="@+id/layout_safe_item_authorizations_container"
+        android:gravity="center"
+        android:orientation="horizontal">
+
+        <ImageView
+            android:layout_width="12dp"
+            android:layout_height="12dp"
+            android:src="@drawable/ic_ether_symbol" />
```

Comment:
thats quite some downscaling here (24 to 12) maybe it makes sense to add a smaller version, might look better on some low res devices